build/convert-sourcetree-to-winui.cmd

```diff
@@ -1,4 +1,6 @@
 @echo off
+setlocal ENABLEEXTENSIONS
+set Platform=
```

Comment:
If you need to set this, this means that a lot of other tooling is going to break, because the "Platform" environment variable is defined at the user or system level and that it will conflict with msbuild's use of it. I'd rather see an error message that tells that this variable should not be defined, so that other types of build will not fail.